inst/templates/tidy_contributing.md

```diff
@@ -0,0 +1,153 @@
+# Contributing to {{{package}}}
+
+-   [Prerequisites](#prerequisites)
+-   [Package mechanics](#package-mechanics)
+    -   [Fork, clone, branch](#fork-clone-branch)
+    -   [Check](#check)
+    -   [Style](#style)
+    -   [Document](#document)
+    -   [Test](#test)
+    -   [NEWS](#news)
+    -   [Re-check](#re-check)
+    -   [Commit](#commit)
+-   [Push and pull](#push-and-pull)
+    -   [Review, revise, repeat](#review-revise-repeat)
+-   [Resources](#resources)
+-   [Code of Conduct](#code-of-conduct)
+
+This explains how to propose a change to {{{package}}} via a pull request using
+Git and Github. 
+
+For more general info about contributing to the tidyverse, see the 
+[Resources](#resources) at the end of this document.
+
+## Prerequisites
```

Comment:
Maybe call this PR process, and make "push and pull" a level three heading?

Reply:
Boom, done 💥! (but like a slow, methodical explosion…)